dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -33,7 +33,7 @@ describe("AppViewComponent", () => {
     namespace: "weee",
   });
 
-  const validProps: IAppViewProps = {
+  const validProps = {
```

Comment:
Ditto (marking now so if I land this I can check these points myself).

Reply:
here is because if I let the `IAppViewProps`, then `validProps.app` may be undefined (according to ts) but if I don't set the type, `validProps.app` is always defined and I don't need to change any test.